### DISCUSSION TITLE
Adding a new /claim type

### DISCUSSION THREAD

**Discussion by sillynapkin** [GriefPrevention/GriefPrevention#1183] [Ideas]
### New Trust Command ###

In an SMP, I made a villager trading farm within my claim
I would like for people to come in freely without a problem, but I am scared that they will grief me.
Because then I would be liked around the SMP and feel very cool and epic

**What I am saying is that there be a new trust called
/VillagerTrust <user>
_or_
/AnimalTrust <user>**

The animal trust would _only let you be allowed to interact with animals (villagers)_
and
The villager trust would _only let you be allowed to trade with villagers_ 
I think that this would be very helpful to me and other people!

Thank you!

**Comment by Jikoo** [comment]
Villager interaction can be configured to be limited by ContainerTrust. Animal interaction is already limited to ContainerTrust, however, this does also allow you to breed and kill animals.
